website/source/docs/guides/acl-migrate-tokens.html.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: "docs"
+page_title: "ACL Token Migration"
+sidebar_current: "docs-guides-acl-migrate-tokens"
+description: |-
+   Consul 1.4.0 introduces a new ACL system with backward incompatible changes.
+   This guide documents how to upgrade the "legacy" tokens after upgrading to
+   1.4.0.
+---
+
+# ACL Token Migration
+
+Consul 1.4.0 introduces a new ACL system with backward incompatible changes.
+This guide documents how to upgrade the "legacy" tokens after upgrading to
+1.4.0.
+
+Since the policy syntax changed to be more precise and flexible to manage, it's
```

Comment:
We may want to re-enforce that legacy tokens _work_ and that you can leave things as they are for any amount of time on the current version and manually migrate as you see fit. When I read these into paragraphs I was a little nervous at how much work it might be.